pkgs/development/libraries/vte/default.nix

```diff
@@ -75,9 +76,11 @@ stdenv.mkDerivation rec {
     systemd
   ];
 
-  propagatedBuildInputs = assert (gtkVersion == "3" || gtkVersion == "4"); [
-    # Required by vte-2.91.pc.
-    (if gtkVersion == "3" then gtk3 else gtk4)
+  # Required by vte-2.91.pc.
+  propagatedBuildInputs = lib.optionals (gtkVersion != null) [
+    (assert (gtkVersion == "3" || gtkVersion == "4");
+    if gtkVersion == "3" then gtk3 else gtk4)
+  ] ++ [
     glib
     pango
```

Comment:
These do not need to be propagated without gtk, which would also make the dev output redundant but I am not sure how to get rid of it without adding complexity of having to repeat it in `buildInputs`.

Reply:
I've just added pangop to buildInputs.

Reply:
We would still want to add glib to not rely on propagation.